applications/nrf5340_audio/src/modules/nrf5340_audio_dk.c

```diff
@@ -1,30 +1,21 @@
 /*
- * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
```suggestion
 * Copyright (c) 2018 Nordic Semiconductor ASA
```

Reply:
Since this is a changed file.

Reply:
This file didn't exist before this, but github seems to believe it is main.c that is renamed.